### PR TITLE
Mark more client mods correctly

### PR DESCRIPTION
**Pull request by TacoMonkey11**
I may have missed a few, but I've marked the majority of them

### CODE REVIEW

File: mods/wthit.pw.toml
Comment (TheStaticVoid) [inline mods/wthit.pw.toml:1]: megane is dependency of wthit, does the removal cause issues for the server load?

**Comment by TheStaticVoid** [comment]
Have you used packwiz to run an server to test client connection with these changes as well?

**Comment by TacoMonkey11** [comment]
I have not, though I will now

File: mods/wthit.pw.toml
Comment (TacoMonkey11) [reply mods/wthit.pw.toml:1]: Marking WTHIT as client only was a mistake, as it does have some features that require a server

**Comment by TacoMonkey11** [comment]
Seems like my packwiz decided to change the hashes of a bunch of random files, not sure why and it causes the update to fail

**Comment by TheStaticVoid** [comment]
use `packwiz refresh` to try and fix it

**Comment by TacoMonkey11** [comment]
I've done that many times and the hashes are not switching back

**Comment by TheStaticVoid** [comment]
You could always undo the commit. Easy if you're using the GitHub desktop client, but I don't know the command if you're using CLI

**Comment by TacoMonkey11** [comment]
I want to figure out the underlying problem first, since I don't want these hashes changing every time I refresh packwiz

**Comment by TacoMonkey11** [comment]
running packwiz refresh on ubuntu linux generates completely different hashes as well

**Comment by TacoMonkey11** [comment]
I'm just going to completely nuke my fork and do this all over again

**Comment by TheStaticVoid** [comment]
https://packwiz.infra.link/tutorials/creating/git/

Might be my fault tbh :tf:

**Comment by TacoMonkey11** [comment]
you have the appropriate .gitattributes file, so I think you are fine

**Comment by TheStaticVoid** [comment]
It's empty 💀

**Pull request closed without merging**